spec/50-privacy.md

```diff
@@ -2,7 +2,7 @@
 
 Requirements to propagate headers to downstream services as well as storing values of these headers opens up potential privacy concerns. Trace vendors MUST NOT use `traceparent` and `tracestate` fields for any personally identifiable or otherwise sensitive information. The only purpose of these fields is to enable trace correlation.
 
-Trace vendors MUST assess the risk of header abuse. This section provides some considerations and initial assessment of the risk associated with storing and propagating these headers. Tracing systems or platforms may choose to inspect and remove sensitive information from the fields before allowing the tracing system to execute code that potentially can propagate or store these fields. All mutations should, however, conform to the list of mutations defined in this specification.
+Trace vendors MUST assess the risk of header abuse. This section provides some considerations and initial assessment of the risk associated with storing and propagating these headers. Trace vendors may choose to inspect and remove sensitive information from the fields before allowing the tracing system to execute code that potentially can propagate or store these fields. All mutations should, however, conform to the list of mutations defined in this specification.
```

Comment:
"Trace vendors" - very limiting in this context. Platforms and proxies may provide this functionality like they do for cookies.

Reply:
I think we should come up with a final and concise wording for that at some point.
Is 'participants' acceptable?

Reply:
In this specific context this change is not correct. As for creating a glossary items - I agree, it is not a bad idea. `Participants` is a good word. Maybe too generic. Do you think "Tracing systems or platforms" is too generic? BTW, we already have a mix of terms like "Tracing vendors " vs. "Tracing systems".

Reply:
I think that "Tracing systems or platform" is too verbose if it is spread all over the spec for every time we mean a system that participates in one way or the other.
Reducing this to one concise phrase is why I am very fond of the glossary section you introduced.

I changed it back to "Tracing systems or platforms" for now. Let's reduce the wording after CR.

Reply:
I like trace participants